frontend/src/pages/vaultSettings/vaultCurrency/constants.ts

```diff
@@ -0,0 +1,70 @@
+import { Fiat } from '../../../model/fiat';
+
+export type CurrencyOption = {
+  value: Fiat;
+  title: string;
+  symbol: string;
+};
+
+export const currencyOptions: CurrencyOption[] = [
+  {
```

Comment:
Can we not hardcode it? Some formatters already give the currency symbols and punctuation.